hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/job/TensorFlowTonyJob.groovy

```diff
@@ -197,4 +197,16 @@ class TensorFlowTonyJob extends HadoopJavaProcessJob implements TensorFlowJob {
     this.workerEnv.put(name, value);
     setJobProperty("worker_env.${name}", value);
   }
+
+  String parseMemoryToMB(String memString) {
```

Comment:
discussed offline. we should probably do the parsing on tony side, so that users can still specify "2g" in azkaban